packages/lib-classifier/src/store/RootStore.js

```diff
@@ -33,6 +34,49 @@ const RootStore = types
   })
 
   .actions(self => {
+    // Private methods
+    function onSubjectReady () {
+      const { classifications, feedback, projects, subjects, workflows, workflowSteps } = self
+      const subject = tryReference(() => subjects?.active)
```

Comment:
[aside] Wait, we have `?.` existence checkers?? This is cool, I didn't realise we had this - where's it coming from? Babel? ES2020?

Reply:
[Optional chaining operator](https://github.com/tc39/proposal-optional-chaining)! It's stage 4 now, so it's part of the default babel support and should be coming to the browser soon!